Dockerfile

```diff
@@ -1,67 +1,77 @@
-FROM quay.io/3scale/docker:ci-2.3.1-3
+FROM centos/ruby-24-centos7
 
-ARG SPHINX_VERSION=2.2.11
-ARG BUNDLER_VERSION=1.12.5
-ARG DB=mysql
+USER root
 
-# Don't use ubuntu mirrors. Rather slow download, than failing build.
-RUN echo "deb http://archive.ubuntu.com/ubuntu precise main restricted universe multiverse\n\
-deb http://archive.ubuntu.com/ubuntu precise-updates main restricted universe multiverse\n\
-deb http://archive.ubuntu.com/ubuntu precise-backports main restricted universe multiverse\n\
-deb http://archive.ubuntu.com/ubuntu precise-security main restricted universe multiverse" > /etc/apt/sources.list
+ARG BUNDLER_ENV
+ENV BUNDLER_ENV="${BUNDLER_ENV}" \
+    TZ=:/etc/localtime \
+    BUNDLE_GEMFILE=Gemfile \
+    BUNDLE_JOBS=5 \
+    BUNDLE_WITHOUT=development:test \
+    NODEJS_SCL=rh-nodejs8
```

Comment:
Now I think system-builder is already setting some of those environments so no need of them